pkg/sensors/base/base.go

```diff
@@ -64,6 +68,27 @@ var (
 	sensorInit sync.Once
 )
 
+func setupExitProgram() {
+	failed := true
+	ks, err := ksyms.KernelSymbols()
+	if err == nil {
+		has_acct_process := ks.IsAvailable("acct_process")
+		has_disassociate_ctty := ks.IsAvailable("disassociate_ctty")
+
+		/* Preffer acct_process over disassociate_ctty */
+		if has_acct_process {
+			Exit.Attach = "acct_process"
+			Exit.Label = "kprobe/acct_process"
+		} else if has_disassociate_ctty {
+			Exit.Attach = "disassociate_ctty"
+			Exit.Label = "kprobe/disassociate_ctty"
+		}
+		failed = !(has_acct_process || has_disassociate_ctty)
```

Comment:
Is there a need to check `has_acct_process` and `has_disassociate_ctty` again?

How about:
```
} else {
   failed = false
}
```

Reply:
yes, I'll change that